src/main.js

```diff
@@ -174,6 +183,24 @@ async function onReady() {
 	try {
 		const [cachedDataDir, nlsConfig] = await Promise.all([nodeCachedDataDir.ensureExists(), resolveNlsConfiguration()]);
 
+		protocol.registerFileProtocol('vscode-file', async (request, callback) => {
+			const uri = new URL(request.url);
+
+			if (uri.pathname.startsWith(app.getAppPath())) {
```

Comment:
This check fails when running a built version. The incoming `request.url` looks like `vscode-file://localhost/Users/alex/src/VSCode-darwin/Code%20-%20OSS.app/Contents/Resources/app/...` and the `app.getAppPath()` looks like `/Users/alex/src/VSCode-darwin/Code - OSS.app/Contents/Resources/app`. The check fails due to the url encoding of ` ` as `%20`. I think this needs to be done somewhat differently to handle url encoding / decoding